src/main/service/GeocodingService.ts

```diff
@@ -0,0 +1,40 @@
+import nodeGeocoder, { Entry, Geocoder, Options } from 'node-geocoder';
+import { GeocodingObject } from '../data/GeocodingObject';
+import * as dotenv from 'dotenv';
+
+/**
+ * A Node library for geocoding our restaurant addresses
+ */
+export class GeocodingService {
+  private geocoder: Geocoder;
+
+  constructor() {
+    dotenv.config();
+    const options: Options = {
+      provider: 'locationiq',
+      apiKey: process.env.GEOCODING_KEY,
+    };
+    this.geocoder = nodeGeocoder(options);
```

Comment:
Just curious, wanted to know why we went with this provider and how we decided this one was better

Reply:
Ah I forgot to put this in the description but I tested out 3 different locations with 3 providers and compared the results to what Google Places API would give us. Then I averaged out the differences and locationiq had the lowest of 0.01880393333.

Reply:
Oh I see, was the free one, OpenMaps or something, not good? We may want to test out more than 3 locations.

Reply:
Looked into locationiq, think its a good provider! Let's stick with this one